package.json

```diff
@@ -32,18 +34,19 @@
     "yargs": "^16.0.3"
   },
   "devDependencies": {
-    "@nomiclabs/buidler": "1.4.5",
-    "@nomiclabs/buidler-ethers": "^2.0.0",
-    "@nomiclabs/buidler-truffle5": "1.3.4",
-    "@nomiclabs/buidler-web3": "1.3.4",
+    "@kyber.network/utils-sc": "2.1.3",
+    "@nomiclabs/hardhat-truffle5": "2.0.0",
+    "@nomiclabs/hardhat-web3": "2.0.0",
+    "@nomiclabs/hardhat-ethers": "2.0.0",
     "@openzeppelin/test-helpers": "0.5.6",
     "chai": "4.2.0",
     "chai-as-promised": "7.1.1",
     "chai-bn": "0.2.1",
     "cp": "^0.2.0",
```

Comment:
Can remove this package I think. Was used for copying files to artifacts folder due to multiple solc versions, but since this repo uses one + hardhat now natively supports multiple solc compilation, dun need anymore